src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```diff
@@ -1,38 +1,40 @@
 .pf-c-clipboard-copy {
   // Clipboard copy toggle
-  --pf-c-clipboard-copy__group-toggle--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-clipboard-copy__group-toggle--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-clipboard-copy__group-toggle--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-clipboard-copy__group-toggle--BorderTopColor: var(--pf-global--BorderColor--300);
-  --pf-c-clipboard-copy__group-toggle--BorderRightColor: var(--pf-global--BorderColor--300);
-  --pf-c-clipboard-copy__group-toggle--BorderBottomColor: var(--pf-global--BorderColor--200);
-  --pf-c-clipboard-copy__group-toggle--BorderLeftColor: var(--pf-global--BorderColor--300);
-  --pf-c-clipboard-copy__group-toggle--hover--BorderBottomColor: var(--pf-global--active-color--100);
-  --pf-c-clipboard-copy__group-toggle--active--BorderBottomWidth: var(--pf-global--BorderWidth--md);
-  --pf-c-clipboard-copy__group-toggle--active--BorderBottomColor: var(--pf-global--active-color--100);
-  --pf-c-clipboard-copy__group-toggle--focus--BorderBottomWidth: var(--pf-global--BorderWidth--md);
-  --pf-c-clipboard-copy__group-toggle--focus--BorderBottomColor: var(--pf-global--active-color--100);
-  --pf-c-clipboard-copy__group-toggle--m-expanded--BorderBottomWidth: var(--pf-global--BorderWidth--md);
-  --pf-c-clipboard-copy__group-toggle--m-expanded--BorderBottomColor: var(--pf-global--active-color--100);
-  --pf-c-clipboard-copy__group-toggle--OutlineOffset: calc(-1 * var(--pf-global--spacer--xs));
+  // --pf-c-clipboard-copy__group-toggle--PaddingRight: var(--pf-global--spacer--md);
+  // --pf-c-clipboard-copy__group-toggle--PaddingLeft: var(--pf-global--spacer--md);
+  // --pf-c-clipboard-copy__group-toggle--BorderWidth: var(--pf-global--BorderWidth--sm);
+  // --pf-c-clipboard-copy__group-toggle--BorderTopColor: var(--pf-global--BorderColor--300);
+  // --pf-c-clipboard-copy__group-toggle--BorderRightColor: var(--pf-global--BorderColor--300);
+  // --pf-c-clipboard-copy__group-toggle--BorderBottomColor: var(--pf-global--BorderColor--200);
+  // --pf-c-clipboard-copy__group-toggle--BorderLeftColor: var(--pf-global--BorderColor--300);
+  // --pf-c-clipboard-copy__group-toggle--hover--BorderBottomColor: var(--pf-global--active-color--100);
+  // --pf-c-clipboard-copy__group-toggle--active--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  // --pf-c-clipboard-copy__group-toggle--active--BorderBottomColor: var(--pf-global--active-color--100);
+  // --pf-c-clipboard-copy__group-toggle--focus--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  // --pf-c-clipboard-copy__group-toggle--focus--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-clipboard-copy__group-toggle--c-button--m-expanded--BorderBottomWidth: var(--pf-global--BorderWidth--md);
```

Comment:
I wonder if we should add a modifier to the button for this, like `pf-m-expanded/selected/active` or something like that, since I imagine we will want to apply these styles in other contexts, too?

Reply:
My take was that the button should just hold the generic hover, active and focus styles, and then components that use the button can add their own modifiers since they're specific to that component, and we can't forsee all of the modifier names. But what is your main reasoning for adding these modifiers to the button I'm open to adding them.

Reply:
The name is the tricky part, but it's a common form control display for active/focus - the thicker, blue bottom border. We would also use it for expanded as the dropdown toggle, too, if we use it there.

Reply:
So are you suggesting adding `pf-m-expanded` to the button since we know we're going to be using that in the clipboard copy at least.

```
  --pf-c-button--m-input--m-expanded--BorderBottomWidth: var(--pf-global--BorderWidth--md);
  --pf-c-button--m-input--m-expanded--BorderBottomColor: var(--pf-global--active-color--100);
```

**pf-m-selected/active:** not sure about these classes because I think the active/focus styles cover these, or am I missing something?

Reply:
Yep, that's it. We can always not since it's only used here for now, and add it later if we end up using it elsewhere (like the dropdown or other components). It just looks like a state this type of button would need. 

Also what's the reasoning behind `pf-m-input` as the modifier name? Is it because the intended use is always going to be "for use in with an input." (as in connected to an input?) I wonder if that's going to end up being limiting.

Reply:
Added the expanded modifier to the button.. @mceledonia do you have any ideas for a name for this button?

Reply:
@christiemolloy Is this for the overall modifier to change styling to input group? If so, maybe something like input-group-button or something specific like that. Maybe something that rolls off the tongue a bit better :)

Reply:
Its for these buttons, thanks for the suggestion!

<img width="399" alt="Screen Shot 2019-07-10 at 10 39 12 AM" src="https://user-images.githubusercontent.com/20118816/60978354-ffe37380-a2fe-11e9-9d65-73000e926ead.png">

Reply:
@mceledonia on second thought, if we use this button in the individual dropdown/select etc. then i dont know if `input-group-button` would be correct, since it wouldn't be part of an input. Maybe a name that refers to the style more so the usage?